### ISSUE TITLE
[Bug] Blank line before/after control structure

### ISSUE THREAD

**Issue by Aliance** [squizlabs/PHP_CodeSniffer#1597]
If I sniffs my code with PSR2 standard, there could be a `Blank line found at start of control structure` violation. For example this code fails:
```php
if (true) {

    // some code here
}
```

But it won't work for this code:
```php
if (true) { // some comment goes here

    // some code here
}
```

I feel this is a bug. A can make a pr with tests, if you approve this issue as a bug.

**Comment by gsherwood** [comment]
This was intentional, but your comment was a good reason to review the rule and see if it should still be like that.

I decided to make the sniff work more like the control signature sniff, which means it will now report errors where it has not reported them before, but I think it aligns better with the other sniffs.

So for your test code, you will now get an error about a blank line being found.

Thanks for the report.
